protect_archiver/downloader/get_motion_event_list.py

```diff
@@ -19,7 +19,8 @@ def get_motion_event_list(
         # path: "/events", method: "GET", query: {start?: timestamp, end?: timestamp, cameras?: list,
         #   smartDetectTypes?: list, type?: string, types?: list, limit?: int, offset?: unknown,
         #   orderDirection?: string, addTimelapseDescriptions?: bool, allCameras?: bool}
```

Comment:
```suggestion
        # TODO: REMARK 2024-Jan-29 @danielfernau #388
        # TODO: The API has been updated and now uses 'type' multiple times instead of a list.
        # TODO: The query parameters documented below are mostly still correct but need to be checked.
        # TODO: Param "withoutDescriptions=true" should be present to avoid unnecessary data in the response.
```